src/mmw/js/src/modeling/views.js

```diff
@@ -326,7 +326,14 @@ var ScenarioTabPanelView = Marionette.ItemView.extend({
     },
 
     renameScenario: function() {
-        var self = this;
+        var self = this,
+            setScenarioName = function(name) {
```

Comment:
Does this need to be defined within `renameScenario`? If not, consider pulling it out.